src/Applications/LDAS_App/ldas_setup

```diff
@@ -1316,13 +1324,28 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '# Format for start/end times is yyyymmdd hhmmss.           #'
     print '#                                                          #'
     print '############################################################'
-    print
+    print '# for adas-coupling exp (COUPLING =1 ) '
+    print '# BEG_DATE needs to be consistent with ${ADAS_exp}'
+    print '# first cycle date (21z, 3z, 9z or 15z) '
+    print '# EXP_DOMAIN needs to be  CS grid as that in ${ADAS_exp}'
```

Comment:
move these help comments down to where COUPLING is defined

Reply:
moved the text to LADAS COUPLING EXP INFO in ldas_setup.